app/grants/management/commands/estimate_clr.py

```diff
@@ -50,12 +50,13 @@ def handle(self, *args, **options):
         if active_clr_rounds:
             for clr_round in active_clr_rounds:
                 print(f"CALCULATING CLR estimates for ROUND: {clr_round.round_num}")
-                predict_clr(
+                a = predict_clr(
                     save_to_db=True,
                     from_date=timezone.now(),
                     clr_round=clr_round,
                     network=network
                 )
+                print(a)
```

Comment:
^ we can undo the changes made for this file

Reply:
done